product.py

```diff
@@ -88,23 +93,32 @@ def get_filterable_attributes(cls):
         return Attribute.search([('filterable', '=', True)])
 
     @classmethod
-    def add_faceting_to_query(cls, search_obj):
+    def _build_es_facets(
```

Comment:
as this method is updating in search_obj only better call this method `_update_es_facets`